2.0.0 FTBFS

With GCC 13.2.1
```
renf_elem/t-get_d.cpp: In function ‘void C_A_T_C_H_T_E_S_T_0()’:
renf_elem/t-get_d.cpp:114:25: error: ‘isinf’ was not declared in this scope; did you mean ‘std::isinf’?
  114 |                 REQUIRE(isinf(floor));
      |                         ^~~~~
```

--- a/doc/news/isinf.rst
+++ b/doc/news/isinf.rst
@@ -1,0 +1,3 @@
+**Fixed:**
+
+* Fixed compilation with GCC 13.2.1.

--- a/libeantic/test/renf_elem/t-get_d.cpp
+++ b/libeantic/test/renf_elem/t-get_d.cpp
@@ -111,24 +111,24 @@ TEST_CASE("Convert renf_elem to double", "[renf_elem][get_d]")
 
             if (renf_elem_cmp_fmpq(a, dmax, nf) < 0)
             {
-                REQUIRE(isinf(floor));
+                REQUIRE(std::isinf(floor));
                 REQUIRE(floor < 0);
             }
             else
             {
-                REQUIRE((!isinf(floor) || floor > 0));
+                REQUIRE((!std::isinf(floor) || floor > 0));
             }
 
             d_get_fmpq(DBL_MAX, dmax);
 
             if (renf_elem_cmp_fmpq(a, dmax, nf) > 0)
             {
-                REQUIRE(isinf(ceil));
+                REQUIRE(std::isinf(ceil));
                 REQUIRE(ceil > 0);
             }
             else
             {
-                REQUIRE((!isinf(ceil) || ceil < 0));
+                REQUIRE((!std::isinf(ceil) || ceil < 0));
             }
 
             fmpq_clear(dmax);

